cdbot/cogs/fun.py

```diff
@@ -166,26 +178,22 @@ def check(reaction, user):
             finally:
                 await sent.delete()
 
-        # React if a message contains an @here or @everyone mention.
-        if any(mention in message.content for mention in ("@here", "@everyone")):
-            await message.add_reaction("\N{SLIGHTLY FROWNING FACE}")
-            await emojify(message, "who pinged")
-
-        # React FBI OPEN UP if message contains trigger words.
-        triggers = ["child", "fbi", "loli", "hentai", "illegal", "maltego"]
-        if any(trigger in message.content.lower() for trigger in triggers):
-            await emojify(message, "fbi open up")
+        if message.content.startswith(":react ") or message.content.startswith(":emojify "):
```

Comment:
This seems like bad practice. Could you not just have it check if it's a valid command like this?
```suggestion
        if !ctx.valid:
```